client/coral-embed-stream/src/Comment.js

```diff
@@ -232,7 +233,7 @@ class Comment extends React.Component {
               removeCommentTag={removeCommentTag}
               showSignInDialog={showSignInDialog}
               reactKey={reply.id}
-              key={`${reply.id}:${depth}`}
+              key={reply.id}
```

Comment:
it can't. I thought so too at first, but the problem was that loadMore would return replies that might already be loaded. Further, even if a comment is a reply to a reply, the deepest nested reply's `parent.id` is the top-level comment.